code/app/src/main/java/com/example/bread/repository/MoodEventRepository.java

```diff
@@ -39,6 +44,12 @@ public void fetchEventsWithParticipantRef(@NonNull DocumentReference participant
     }
 
     public void addMoodEvent(@NonNull MoodEvent moodEvent, @NonNull OnSuccessListener<Void> onSuccessListener, OnFailureListener onFailureListener) {
+// TODO: newly added
+//        FirebaseUser currentUser = FirebaseAuth.getInstance().getCurrentUser();
+//        if (currentUser != null) {
+//            moodEvent.setParticipantRef(currentUser.getUid()); // Set the participant reference to the current user's UID
+//        }
+
```

Comment:
why is this todo here? plus participant ref is not the Uid fyi. Please check out the firebase details.